config_files/initialization_config.yml

```diff
@@ -19,14 +25,24 @@
                   22                    # number of points in the sequence
   ]
 
-  sig_b_coeff_power: -6.8               # the power for the coefficient of the sigma_bs = coef *L^2,
-                                        # will give coef = 10^(sig_b_coeff_power)
   #####################################################################################################################
 
+  #####################################################################################################################
+  # Log-linear regression#
+  ########################
+  # Target strenth (TS) - length (L) regression: TS=m*log10(L)+b
+  TS_length_regression_parameters:
+    pacific_hake:                         # corresponding species text code
+      number_code: 22500                  # species number code
```

Comment:
I wonder if it'll be good to have another place where the species name and code mapping is provided. It seems should be separate from the TS-length relationship and the config yamls.

Reply:
Actually now thinking about it, maybe there could be a place where the various type of codes, like the code for different sex could also be put into that other place...

Reply:
I think these can be in `core.py` along with other definitions:
```python
SEX_CODE_TEXT = {
   "male": "M",
   "female": "F",
   "unsexed": "U",  # or just an empty string "", depending on what's in the spreadsheet
}

SEX_CODE_TEXT = {
   "male": 1,
   "female": 2,
   "unsexed": 3, 
}

SPECIES_CODE = {
   "hake": 22500,
   "krill": ...,
}
```